samples/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This sample application provides an example usage of the display ILI9163C from ILITEK.
+The example includes PWM control for backlight dimming and displays text and a counter on the screen.
+
+- Displays "6TRON BY CATIE!" on the screen. 
+- Shows an incrementing counter at the bottom of the screen. 
+- Uses PWM to control the backlight brightness in a pulsing pattern. 
+- Refreshes the display content periodically using LittlevGL (LVGL).
+
+> [!NOTE]
+>  Ensure the display and PWM hardware configurations match the setup below for correct operation.
+
+# Requirements
+
+- Hardware: Zest Display LCD shield with ILI9163C display and any compatible Zest Core board.
+- Board allowing I2C communication.
+- Configuration Options:
+- `CONFIG_DISPLAY=y` in prj.conf to use DISPLAY API.
+- `CONFIG_PWM=y` in prj.conf to use PWM API.
+
+# References
+
+- [ER-TFT018-2 LCD](https://www.buydisplay.com/download/manual/ER-TFT018-2_Datasheet.pdf).
+- [ILI9163C Datasheet](https://www.buydisplay.com/download/ic/ILI9163.pdf).
+
+# Building and Running
+
+```shell
+cd <driver_directory>
+west build -p always -b <BOARD> samples/
+west flash
+```
+
+# Sample Output
+
```

Comment:
I don't know if it is a good practice but a screen of the display might be wanted here.

Reply:
We must ask to @clementlgl